apps/web/app/s/[surveyId]/layout.tsx

```diff
@@ -1,6 +1,24 @@
 import LegalFooter from "@/app/s/[surveyId]/components/LegalFooter";
+import { type Metadata } from "next/types";
 
-export default function SurveyLayout({ children }) {
+export const metadata: Metadata = {
+  title: "Formbricks",
+  description: "Open-Source In-Product Survey Platform",
```

Comment:
Change to "Create your own survey like this with Formbricks' open source survey suite."

Reply:
![Screenshot from 2023-10-18 16-24-43](https://github.com/formbricks/formbricks/assets/92802904/1bd32b25-6b15-4d89-9c29-c8f99a82b888)
Done..